src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -295,8 +339,10 @@ export default class WaypointModel {
         this._positionModel = null;
 
         this.isHold = false;
-        this.speedRestriction = -1;
-        this.altitudeRestriction = -1;
+        this.speedMaximum = -1;
```

Comment:
we should use `INVALID_VALUE` here as well.  perhaps add that to a constant somewhere and give it a more descriptive name like `INVALID_NUMBER_VALUE` or soething